### PR TITLE
Zaki/update_gravity_bridge

### PR DESCRIPTION
**Pull request by zmanian**
update to gravity bridge master

### CODE REVIEW

File: steward/Cargo.toml [hunk 1]
```diff
@@ -13,9 +13,9 @@ chrono = "0.4"
 clap = "3.0.0-beta.5"
 clarity = "0.4.12"
 deep_space ={ git = "https://github.com/iqlusioninc/deep_space/", branch = "master" }
-ethers = { version = "0.6.1", features = ["abigen"] }
+ethers = { git = "https://github.com/gakonst/ethers-rs.git", branch="zaki/error_abi_support", features = ["abigen"] }
```
Comment (EricBolten) [inline steward/Cargo.toml]: If you're intending to use your forked version, this needs to be:

`ethers = { git = "https://github.com/iqlusioninc/ethers-rs.git", branch="zaki/error_abi_support", features = ["abigen"] }`

File: steward_abi/Cargo.toml [hunk 2]
```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-ethers = { version = "0.6.1", features=["abigen"] }
+ethers = { git = "https://github.com/gakonst/ethers-rs.git", branch="zaki/error_abi_support", features = ["abigen"] }
```
Comment (EricBolten) [inline steward_abi/Cargo.toml]: Same here:

`ethers = { git = "https://github.com/iqlusioninc/ethers-rs.git", branch="zaki/error_abi_support", features = ["abigen"] }`

File: steward/src/commands/orchestrator/start.rs [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+            let config = APP.config();
+            let cosmos_prefix = config.cosmos.prefix.clone();
+
+            let cosmos_key = config.load_deep_space_key(self.cosmos_key.clone());
+            let cosmos_address = cosmos_key.to_address(&cosmos_prefix).unwrap();
+
+            let ethereum_wallet = config.load_ethers_wallet(self.ethereum_key.clone());
+            let ethereum_address = ethereum_wallet.address();
+
+            let contract_address: ethers::prelude::H160 = config
```
Comment (EricBolten) [inline steward/src/commands/orchestrator/start.rs]: Elsewhere we've imported this with `use ethers::types::Address as EthAddress`

File: steward/src/commands/orchestrator/start.rs [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    #[clap(short = 'e', long)]
+    ethereum_key: String,
+
+    #[clap(short, long)]
+    orchestrator_only: bool,
+}
+
+impl Runnable for StartCommand {
+    fn run(&self) {
+        {
```
Comment (EricBolten) [inline steward/src/commands/orchestrator/start.rs]: I think you've wrapped this in an extra layer of brackets.

**Review by EricBolten — COMMENTED** [review]
Some suggested fixes.

File: steward/Cargo.toml (hunk 1 above)
Comment (zmanian) [reply steward/Cargo.toml]: Thank you

File: steward_abi/Cargo.toml (hunk 2 above)
Comment (zmanian) [reply steward_abi/Cargo.toml]: ty

**Review by EricBolten — APPROVED** [review]
LGTM

**Pull request merged**